src/stories/Library/Lists/list-description/list-description.scss

```diff
@@ -27,7 +27,11 @@ dl.list-description {
     flex-direction: column;
   }
 
-  .link-tag {
+  // TODO: Reevaluate this when implementing logic to determine if 'link-tag' styled
+  // values in dpl-react should be 'a' tags or 'span' tags as currently implemented.
+  .link-tag,
```

Comment:
As I see it we should drop using *and* supporting `link-tag` for description elements that are not links.

Using nested elements is against our BEM structure and it is just odd to have something that looks like an interactive element but is not.

I think you need to clear this with the LSBP team before proceeding.